clarifai/models/model_serving/README.md

```diff
@@ -142,6 +142,8 @@ Upload
 
 ```bash
 $ clarifai upload model --url <url> --user-app <your_user_id>/<your_app_id> --id <your_model_id>
+# or upload built file directly
```

Comment:
I think we should put this on top and url as the second option

Reply:
Cause I believe most users will use direct upload. A very small minority will be interested in url